server/svix-server/src/v1/endpoints/endpoint/mod.rs

```diff
@@ -160,6 +163,19 @@ pub struct EndpointIn {
     pub key: Option<EndpointSecret>,
 }
 
+impl EndpointIn {
+    pub fn take_or_generate_key(
```

Comment:
```suggestion
    pub fn key_take_or_generate(
```
So that it autocompletes nicely in the editor when you type `key`